server/models/permission.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, DataTypes) => {
+  const Permission = sequelize.define('Permission', {
+    access: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+  }, {});
+
+  Permission.associate = (models) => {
```

Comment:
So users don't have permissions directly, users have roles and roles have permissions

Reply:
should I create a joint table between the user and the permissions?